docs/development/design/new-sphinx-guides.rst

```diff
@@ -111,33 +112,110 @@ highly recommended.
       -  Rebuild the HTML docs ``make html`` and notice that now the
          times are printed (yay!)
 
-5. Writing narrative documentation with Sphinx
+6. Writing narrative documentation with Sphinx
 
    -  First focus on ``index.md``, diving more into Markdown
       and mentioning Semantic Line Breaks.
    -  Then add another ``.md`` file to teach how ``toctree`` works.
    -  Then continue introducing elements of the syntax to add pictures,
       cross-references, and the like.
 
-6. Describing code in Sphinx
+7. Describing code in Sphinx
 
    -  Explain the Python domain as part of narrative documentation to
       interleave code with text, include doctests, and justify the
       usefulness of the next section.
 
-7. Autogenerating documentation from code in Sphinx
-8. Deploying a Sphinx project online
+8. Autogenerating documentation from code in Sphinx
+9. Deploying a Sphinx project online
 
    - A bit of background on the options: GitHub/GitLab Pages,
      custom server, Netlify, Read the Docs
    - Make reference to Read the Docs tutorial
 
-9. Appendix: Using Jupyter notebooks inside Sphinx
+10. Appendix: Using Jupyter notebooks inside Sphinx
+11. Appendix: Where to go from here
+
+   - Refer the user to the Sphinx, reST and MyST references,
+     prominent projects already using Sphinx,
+     compilations of themes and extensions,
+     the development documentation.
 
 .. note 
 
    - Looks like MathJax is enabled by default now? Can't see a reference in the docstrings
 
+Read the Docs tutorial
+----------------------
+
+1. What is Read the Docs
```

Comment:
Not "The Read the Docs Way"? :D

![](https://i.ebayimg.com/images/g/ZuUAAOSwgo9d~DaS/s-l400.jpg)

Reply:
😃